apps/gitness/src/pages-v2/pull-request/hooks/usePRChecksDecision.ts

```diff
@@ -0,0 +1,181 @@
+import { useEffect, useMemo, useState } from 'react'
+
+import pluralize from 'pluralize'
```

Comment:
We probably don't need `pluralize` for our needs. It's 12k of added size to conditionally add `s`. I'd say that we can offer a smaller solution and use our i18n stuff (which we ned to do anyway).

```typescript
function easyPluralize(count: number, singular: string, plural: string, include?: boolean): string {
  const word = count === 1 ? singular : plural

  return include ? `${count} ${word}` : word
}

// ...

console.log('I saved', easyPluralize(12, 'KB', 'KBs', true)) // "I saved 12 KBs"
```

Reply:
Had thought about it as well, but this handles only trivial cases. For e.g.

`thief` needs to be changed to `thieves` and here the change is more than just adding an additional character.

Reply:
put a todo for now